signer/tuf_on_ci_sign/_signer_repository.py

```diff
@@ -100,6 +100,11 @@ def _find_changed_roles(known_good_dir: str, signing_event_dir: str) -> list[str
 class SignerRepository(Repository):
     """A repository implementation for the signer tool"""
 
+    """Maximum depth within role directory. This
+    should be the same as CIRepository.MaxDepth
+    in the repo project"""
+    MaxDepth: int = 4
+
```

Comment:
and add import: 
```python
from tuf_on_ci._repository import CIRepository
```

Reply:
it would be better to have this be in a configuration module that both modules import
like
```python
from tuf_on_ci import config
config.MAX_TARGETS_DEPTH
```

Reply:
`_*` prefixes are supposed to indicate protected/private objects but nothing in python enforces that

Reply:
@mrjoelkamp I was looking for something shared, but it looks like these projects are peers in the mono repo. How does this work?